index.html

```diff
@@ -3,10 +3,162 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,400;0,700;0,900;1,400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Gothic+A1:wght@700;900&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>Ili Artes - crafts</title>
   </head>
   <body>
+    <header>
+      <div id="submenu-container">
+        <ul id="submenu">
+          <li class="submenu-item social">
+            <img src="./assets/icons/whiteFB.png" alt="" />
+          </li>
+          <li class="submenu-item social">
+            <img src="./assets/icons/Twitter_bird.png" alt="" />
+          </li>
+          <li class="submenu-item">English</li>
+          <li class="submenu-item">My Page</li>
+          <li class="submenu-item">Logout</li>
+        </ul>
+      </div>
+      <nav>
+        <img
+          id="menu-btn"
+          src="./assets/icons/menu-btn.png"
+          alt="menu button"
+        />
+      </nav>
+      <ul id="menu">
+        <li id="logo-container">
+          <a id="logo" href="./index.html"
+            ><img src="./assets/Mobile/about/Logo.svg" alt=""
+          /></a>
+        </li>
+        <li class="menu-item"><a href="./about.html">About</a></li>
+        <li class="menu-item"><a href="/">Program</a></li>
+        <li class="menu-item"><a href="/">Join</a></li>
+        <li class="menu-item"><a href="/">Sponsor</a></li>
+        <li class="menu-item"><a href="/">News</a></li>
+        <li class="menu-item"><a href="./index.html">CC Campain</a></li>
+      </ul>
+      <div id="logo-container-mobile">
+        <a id="logo-mobile" href="./index.html"
+          ><img src="./assets/Mobile/about/Logo.svg" alt=""
+        /></a>
+      </div>
+      <img src="./assets/icons/menu-close.svg" alt="" />
+    </header>
```

Comment:
Greeting!

According to the previous reviewer, kindly I suggest you stick to the design as much as possible by following: `padding, margin, font, colors etc.`
- It seems like your design is still break on the mobile version, kindly I suggest you update it to make your project look more professional.

> ### The Design
>
><img width="375" alt="Screenshot 2023-08-02 at 10 33 54 PM" src="https://github.com/Estete9/IliArtes/assets/91237525/1468472f-2c7e-4fbe-9383-3513deef1a75">
>
> ### Your Design 
>
><img width="375" alt="Screenshot 2023-08-02 at 10 31 01 PM" src="https://github.com/Estete9/IliArtes/assets/91237525/19583296-f86c-463b-bbf7-8bc4790de3ee">

Reply:
Hello my friend, can you give me more feedback? I just ran the page and I can't reproduce de error.

This is a screenshot of the app running
<img width="340" alt="image" src="https://github.com/Estete9/IliArtes/assets/23634070/4203f65d-50c6-4d35-996b-7920b2abc0e5">